pkg/api/volume/schema.go

```diff
@@ -0,0 +1,32 @@
+package volume
+
+import (
+	"net/http"
+
+	"github.com/rancher/apiserver/pkg/types"
+	"github.com/rancher/steve/pkg/schema"
+	"github.com/rancher/steve/pkg/server"
+	"github.com/rancher/wrangler/pkg/schemas"
+
+	"github.com/harvester/harvester/pkg/config"
+)
+
+func RegisterSchema(scaled *config.Scaled, server *server.Server, options config.Options) error {
+	t := schema.Template{
+		ID: "harvesterhci.io.exportvolume",
```

Comment:
this doesn't look like a valid resource ID

Reply:
@gitlawr updated